nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-process-group-configuration.js

```diff
@@ -272,6 +274,25 @@
                     $('#process-group-default-back-pressure-object-threshold').removeClass('unset').val(processGroup.defaultBackPressureObjectThreshold);
                     $('#process-group-default-back-pressure-data-size-threshold').removeClass('unset').val(processGroup.defaultBackPressureDataSizeThreshold);
 
+                    $('#process-group-log-to-own-file-combo').removeClass('unset').combo({
+                        options: [{
+                            text: 'Yes',
+                            value: true,
+                            description: 'Logging will be generated to dedicated log file for the process group.'
```

Comment:
"Messages logged by components in this group will be sent to the standard app log, as well as a separate log file specific to this group."?